ci-operator/step-registry/cucushift/installer/check/aws/byo-kms-key/cucushift-installer-check-aws-byo-kms-key-commands.sh

```diff
@@ -0,0 +1,156 @@
+# "Value": "ci-op-f59tc36y-a3cdc-vxf25-dynamic-pvc-9d10a326-ec97-42f0-88c9-6e3f0c65402d"
+monitoringpvc_kms_key=$(cat ${ARTIFACT_DIR}/compute_vols.json | jq -r '.Volumes[] | select(any(.Tags[]; .Key == "Name" and (.Value | contains("dynamic-pvc")))) | .KmsKeyId' | sort | uniq)
+
+echo "controlPlane keys: $(show_key "${control_plane_kms_key}")"
+echo "compute keys: $(show_key "${compute_kms_key}")"
+echo "monitoringpvc keys: $(show_key "${monitoringpvc_kms_key}")"
+
+
+echo "-------------------------------------------------------------"
+echo "KMS keys configured in install-config.yaml"
+echo "-------------------------------------------------------------"
+
+ic_platform_key=$(yq-go r "${CONFIG}" 'platform.aws.defaultMachinePlatform.rootVolume.kmsKeyARN')
+ic_control_plane_key=$(yq-go r "${CONFIG}" 'controlPlane.platform.aws.rootVolume.kmsKeyARN')
+ic_compute_key=$(yq-go r "${CONFIG}" 'compute[0].platform.aws.rootVolume.kmsKeyARN')
+
+echo "platform: $(show_key "${ic_platform_key}")"
+echo "controlPlane: $(show_key "${ic_control_plane_key}")"
+echo "compute: $(show_key "${ic_compute_key}")"
+
+
+
+echo "-------------------------------------------------------------"
+echo "Expected keys"
+echo "-------------------------------------------------------------"
+
+expected_control_plane_key="${default_kms_key_arn}"
+expected_compute_key="${default_kms_key_arn}"
+
+if ! is_empty "$ic_platform_key"; then
+  echo "platform.aws.defaultMachinePlatform.rootVolume.kmsKeyARN was found: $(show_key "${ic_platform_key}")"
+  expected_control_plane_key="${ic_platform_key}"
+  expected_compute_key="${ic_platform_key}"
+fi
+
+if ! is_empty "$ic_control_plane_key"; then
+  echo "controlPlane.platform.aws.rootVolume.kmsKeyARN was found: $(show_key "${ic_control_plane_key}")"
+  expected_control_plane_key="${ic_control_plane_key}"
+fi
+
+if ! is_empty "$ic_compute_key"; then
+  echo "compute[0].platform.aws.rootVolume.kmsKeyARN was found: $(show_key "${ic_compute_key}")"
+  expected_compute_key="${ic_compute_key}"
+fi
+
+echo "expected_control_plane_key: $(show_key "${expected_control_plane_key}")"
+echo "expected_compute_key: $(show_key "${expected_compute_key}")"
+
+
+echo "-------------------------------------------------------------"
+echo "Checking"
+echo "-------------------------------------------------------------"
+
+if [[ "${control_plane_kms_key}" != "${expected_control_plane_key}" ]]; then
+  echo "FAIL: KMS key: control plane node: expect: $(show_key "${expected_control_plane_key}"), current: $(show_key "${control_plane_kms_key}")"
+  ret=$((ret+1))
+else
+  echo "PASS: KMS key: control plane node: $(show_key "${expected_control_plane_key}")"
+fi
+
+if [[ "${compute_kms_key}" != "${expected_compute_key}" ]]; then
+  echo "FAIL: KMS key: compute node: expect: $(show_key "${expected_compute_key}"), current: $(show_key "${compute_kms_key}")"
+  ret=$((ret+1))
+else
+  echo "PASS: KMS key: compute node: $(show_key "${expected_compute_key}")"
+fi
+
```

Comment:
pvc kms key checking has a pre-condition, have to enable `ENABLE_AWS_KMS_KEY_DEFAULT_MACHINE`, I guess we need to add a condition here to run the following checking, similar to [azure](https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/check/azure/des/cucushift-installer-check-azure-des-commands.sh#L140), maybe we even need to judge OCP version.